### ISSUE TITLE
parameter setting and other experience

### ISSUE THREAD

**Issue by tangeroo** [yangyanli/PointCNN#177]
Hi,
 Thanks for your sharing,it really helps.The ideas in this net are really great.
  I wonder which parameters I may set again if I want to use this net in a new situation.and what's more,can you share some experiences in the process of giving out this wonderful net?Thank you.
 I'm new hand in this field,forgive me for any unreasonable request.

**Comment by burui11087** [comment]
Hi @tangeroo 
Batch size, learning rate, xconv_params and fc_params is important. If you find that network is overfitting, just reduce the channel numbers in xconv_params or increase the dropout rate in FC layer. tuning sampling , epsilon or adding more information such as RGB is also work.
Generally, if your task is classification, increase the receptive filed by tuning 'K', 'D' in xconv_params and segmentation task is opposite.

Thanks

**Comment by smgrz** [comment]
Hi,
I'm doing some classification on my own dataset and I've realised, when I change following settings: rotation_range = [0, math.pi, 0, 'u'] to rotation_range = [0, math.pi/18, 0, 'g'] I'm getting much better results.
I've tried to understand (from code) what does it mean but unfortunately I wasn't successful. 
Could your please give me some hints or further links where I can find more information (or comments) about the settings parameter.

Thanks

**Comment by dandanaus** [comment]
@smgrz did you have any luck with the settings? I am getting overfitting and trying to see how i can make some of the changes

**Comment by dandanaus** [comment]
> If you find that network is overfitting, just reduce the channel numbers in xconv_params or increase the dropout rate in FC layer

@burui11087 can you please provide some more info on the segmentation side of things if we are seeing overfitting?
